README.md

```diff
@@ -200,3 +200,59 @@ You can run Celery workers for Concent with:
 ``` bash
 concent_api/celery worker --app concent_api --loglevel info --queues concent,conductor,verifier
 ```
+
+### Deploying GNTDeposit contract
+
+To be able to run e2e tests locally, that contains transactions on blockchain,
+Concent needs separate GNTDeposit contract to work without errors.
+
+1. Get [GNTDeposit.sol](https://github.com/golemfactory/golem-contracts/blob/master/contracts/GNTDeposit.sol)
+from `golem-contracts` repository.
+
+2. Fallow this [tutorial](https://medium.com/swlh/deploy-smart-contracts-on-ropsten-testnet-through-ethereum-remix-233cd1494b4b)
+to deploy GNTDeposit.sol on `rinkeby` ethereum testnet.
+
+    It involves installing `MetaMask` on your browser, creating ethereum account with some testnet ETH and copy
+    GNTDeposit.sol with all dependencies to single file in solidity IDE - `remix`. You can use
+    [solidity-flattener](https://github.com/BlockCatIO/solidity-flattener) to do so.
```

Comment:
`on your browser` -> `in your browser`
`copy` -> `copying`
`GNTDeposit.sol` -> `` `GNTDeposit.sol` ``

Reply:
Done